compiler/dcalc/implementing defaults

```diff
@@ -0,0 +1,7 @@
+implementing options translation
+
+practical issues:
+ * bindlib code is hard to read without let+.
```

Comment:
This should be cleaned when PR is ready for merge, the info transferred into comments in the code at the right places.

Reply:
fixed